packages/contact-center/station-login/src/station-login/index.tsx

```diff
@@ -7,15 +7,16 @@ import StationLoginPresentational from './station-login.presentational';
 import {useStationLogin} from '../helper';
 
 const StationLogin: React.FunctionComponent = observer(() => {
-  const {loginState, setLoginState, ccSdk, isAvailable} = store;
-  const result = useStationLogin();
+  const {cc, teams, loginOptions, loginReqParam, setDeviceType, setDialNumber, setTeam} = store;
+  const result = useStationLogin({cc, loginReqParam});
 
   const props = {
     ...result,
-    loginState,
-    setLoginState,
-    ccSdk,
-    isAvailable,
+    teams,
+    loginOptions,
+    setDeviceType,
+    setDialNumber,
+    setTeam
```

Comment:
Please check if we can get these from helpers with `useState` hook

Reply:
Addressed. Thanks for the suggestion